urdf/include/urdf/model.h

```diff
@@ -41,7 +41,7 @@
 #include <map>
 #include <memory>
 
-#include "tinyxml.h"  //NOLINT
+#include <tinyxml.h>
```

Comment:
Ok, I thought that was related to the " instead of the '>'. I'll put the `NOLINT` comment back in place.